src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -195,13 +200,29 @@ def _execute(self, reqRecords):
             totalNumRequests += 1
             if self._checkClean(wflow):
                 cleanNumRequests += 1
+                cleanReqList.append(wflow['RequestName'])
```

Comment:
Should I assume this `cleanReqList` will also contain those requests flagged as ForceArchive=True?

Reply:
No, they are accounted separately. And because we are speaking about ACDCs here, that's why those are checked explicitly for there type and accumulated in a separate counter as `Resubmissions`